_projects/2024/100541120/100541120.Rmd

```diff
@@ -0,0 +1,997 @@
+    panel.grid = element_blank(),
+    axis.text.y = element_text(size = 13, family = "mulish", color = "#0A0A0A"), 
+    # Remove the grid lines
+    panel.grid.major = element_blank(),
+    # Hide the legend
+    legend.position = "none",
+    # Adjust plot margins
+    plot.margin = margin(t = 0, r = 5, b = 1, l= 2), 
+    # Remove panel and plot backgrounds
+    panel.background = element_blank(), 
+    plot.background = element_blank(),
+    # Style the caption with markdown support
+    plot.caption = element_markdown(hjust = 0, vjust = 1, face = "plain", 
+                                    size = 8, family = "mulish", margin = margin(t = 0, r = 0, b = 0, l = 11)),
+    # Style the title and subtitle with specific positions and sizes
+    plot.title = element_text(hjust = 0.07, vjust = 0, 
+                              size = 29, family = "mulish"),# Adjust bottom margin for title
+    plot.subtitle = element_text(hjust = 0.12, vjust = 0, 
+                                 size = 22, color = "#353939", family = "mulish"),# Adjust bottom margin for subtitle
+    # Position title and subtitle within the plot panel
+    plot.title.position = "plot",
+    plot.subtitle.position = "plot"
+  )
+
+barchart_coc1
+```
+
+
+
+Now I can continue with the flags, which have been one of the main challenges. First, it was difficult to find the right package for this task. Second, the issue was with the images themselves, as I initially had to download them, which added to the complexity.
+
+Finally, I decided to use a package that I have downloaded from Github, called "flagon". <"https://github.com/coolbutuseless/flagon">
+
+But there is also a particularity, some of the flags of the original plot are not in this package (because these are from regions inside a country), so I needed to download in the project file.
+
+
+```{r, fig.width= 8.5, fig.height= 6}
+#Install the package:
+library(devtools)
+install_local("C:/Users/jorge/Desktop/CSS/Data-Visualization/Cocaine in Europe - Datavis/flagon-master")
+
+
+library(png)
+library(flagon)
+# Read PNG images for flags
+Australia <- readPNG(flagon::flags("au"))
+Albania <- readPNG(flagon::flags("al"))
+Estados_Unidos <- readPNG(flagon::flags("us"))
+Escocia <- readPNG("C:/Users/jorge/Desktop/CSS/Data-Visualization/Cocaine in Europe - Datavis/flagon-master/inst/png/sct.png")
+Inglaterra <- readPNG("C:/Users/jorge/Desktop/CSS/Data-Visualization/Cocaine in Europe - Datavis/Images/England and Wales.png")
+España <-  readPNG(flagon::flags("es"))
+Paises_Bajos <- readPNG(flagon::flags("nl"))
+Irlanda_norte <- readPNG("C:/Users/jorge/Desktop/CSS/Data-Visualization/Cocaine in Europe - Datavis/Images/Northern Ireland.png")
+Uruguay <- readPNG(flagon::flags("uy"))
+Argentina <- readPNG(flagon::flags("ar"))
+Irlanda <- readPNG(flagon::flags("ie"))
+Canada <- readPNG(flagon::flags("ca"))
+Polonia <- readPNG(flagon::flags("pl"))
+Chile <- readPNG(flagon::flags("cl"))
+CostaRica <- readPNG(flagon::flags("cr"))
```

Comment:
You only need 3 special flags not included in flagon, right? Then remove the rest of them from the "Flags" directory. And those 3 should be just

```r
Inglaterra <- readPNG("Flags/blablabla.png")
```

etc.